docs/CONTRIBUTING.md

```diff
@@ -421,3 +426,4 @@ If your PR doesn't need to be included in the changelog, please start the commit
 [npm-install-docs]: https://docs.npmjs.com/downloading-and-installing-node-js-and-npm
 [poetry-website]: https://python-poetry.org/
 [poetry-docs]: https://python-poetry.org/docs/
+[upgrading-file]: ../UPGRADING
```

Comment:
```suggestion
[upgrading-file]: UPGRADING.md
```
This allows the link to work when viewing the documentation via the GitHub web interface, which is especially likely with CONTRIBUTING.md, since GitHub prompts contributors to read it there.